### PR TITLE
[DEVOPS-1216] iohk nix 18.09 update

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -24,7 +24,7 @@ let
         sha256 = "0xfwyh21x6r2x7rjgf951gkkld3h10x05qr79im3gvhsgnq3nzmv";
       };
     in (import "${nixopsUnstable}/release.nix" {
-         nixpkgs = localLib.fetchNixPkgs;
+         inherit (localLib) nixpkgs;
```
Comment (jbgi) [inline default.nix]: should this be
`nixpkgs = pkgs`
?

**Comment by disassembler** [comment]
I'm marking DO NOT MERGE as we don't want this to hit master until release is completed.

File: benchmarks/shell.nix [hunk 2]
```diff
@@ -1,5 +1,5 @@
 { localLib ? import ../lib.nix
-, pkgs ? import (localLib.fetchNixPkgs) {}
+, pkgs ? localLib.nixpkgs) {}
```
Comment (cleverca22) [inline benchmarks/shell.nix]: is that `)` in the right place?

**Comment by disassembler** [comment]
bors ping

**Pull request merged**